scripts/build_gh_pages

```diff
@@ -29,25 +29,38 @@ if [[ -n "$(ls "${TARGET_DIR}"/*)" ]]; then
   exit 1
 fi
 
-# Generate docs directly in the target dir.
-cargo doc --no-deps --target-dir="${TARGET_DIR}"
+readonly CURRENT_DIR="$(pwd)"
 
-# Remove non-doc artifacts from the target dir.
-rm --recursive --force "${TARGET_DIR}/debug"
+paths=("." "oak_abi" "oak_utils")
```

Comment:
Could do with either a comment and/or better names for these arrays, to indicate what they're supposed to hold.  (Maybe `source_paths` / `target_subdirs` / `titles` ?)